citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -141,11 +145,36 @@ $(function() {
 		var authId = $(this).attr('data-authority-id');
 		$("#uriEditorLocalId").val(authId);
 		$("#editorAuthorityUsed").html("Using stored authority entry <i>" + $(this).attr('data-authority-name') + "</i>.");
+		showPersonNameInModal($(this).attr('data-authority-name'), "Editor");
 		$("#uriLoadingFoundEditor").popover('hide');
 		event.preventDefault();
 	});
 });
 
+function showPersonNameInModal(name, personType){
+	var parsedName = name;
+	if(name.includes("(") && name.includes(")")) {
+		parsedName = name.substring(0, name.indexOf("(")) + name.substring(name.indexOf(")")+1);
+	}
+	if(name.includes("(")) {
+		parsedName = name.substring(0, name.indexOf("("));
+	}
+	if(parsedName.split(",").length -1 > 1) {
```

Comment:
why not length > 2?

Reply:
We are ignoring anything that comes after the second comma. So there are more than 1 comma, cut the string until the second comma occurs.

Reply:
length - 1 > 1 or length > 2 is the same isn't it? just less complicated.